lib/tiki/torch/config.rb

```diff
@@ -44,6 +44,10 @@ class SerializationStrategies
       # handler in a lambda.
       attribute :publishing_error_handler, Object, default: lambda { |_, _| Proc.new { |_, _, _| } }
 
+      attribute :retry_interval_secs, Integer
+      attribute :retry_timeout_secs, Integer
+      attribute :retry_size, Integer
```

Comment:
Maybe `retry_count` would be better?

Reply:
renamed it to `retry_Size` to `retry_count`.